basics.js

```diff
@@ -1,24 +1,70 @@
 /* Create a `myName` variable and assign it a String value */
+var myName = "Dev League";
 
 /* Create a `person` variable and give it 2 properties,
  * `name`, assign it the same name as before,
  * as well as an `age` (number);
  */
+var person = {name: myName, age: 60};
 
 /* Create a function called `greet`,
  * it should take a 1 parameter, `name`
  * and it should print "Hello, my name is {name}"
  */
+function greet(name)
+  {
+    console.log('Hello, my name is ' + name);
+  }
 
+greet(person.name);
```

Comment:
No need to individually call the function in `basic.js`, that's done by the tests already.